### PR TITLE
include docs snippets tests in bk and fix broken tests

### PR DESCRIPTION
**Pull request by yuhan**
## Summary

This was totally an oversight. I found out we did not include tests for example/docs_snippets in bk...😅

## Test Plan
bk

### CODE REVIEW

**Review by sryza — APPROVED** [review]
Yikes.  Thanks for fixing all of these.

File: .buildkite/dagster-buildkite/dagster_buildkite/steps/dagster.py [hunk 1]
```diff
@@ -75,6 +75,22 @@ def dbt_example_extra_cmds_fn(_):
     ]
 
 
+def docs_snippets_extra_cmds_fn(_):
+    return [
+        "pushd examples/docs_snippets",
+        # Run the postgres db. We are in docker running docker
+        # so this will be a sibling container.
+        "docker-compose up -d --remove-orphans",  # clean up in hooks/pre-exit
```
Comment (alangenfeld) [inline .buildkite/dagster-buildkite/dagster_buildkite/steps/dagster.py:83]: there is supposed to be a corresponding addition in `.buildkite/hooks/pre-exit` - i guess this comment needs to be clearer and louder

Comment (alangenfeld) [reply .buildkite/dagster-buildkite/dagster_buildkite/steps/dagster.py:83]: that said, what docs snippets need postgres? seems odd that we need this

Comment (yuhan) [reply .buildkite/dagster-buildkite/dagster_buildkite/steps/dagster.py:83]: in a mode/resource tutorial, we mentioned postgres so we have tests covering that

**Pull request merged**